apis/logging/v1/kibana_types.go

```diff
@@ -13,19 +13,27 @@ type KibanaSpec struct {
 	//
 	ManagementState ManagementState `json:"managementState"`
 
-	// The resource requirements for Kibana
+	// Resource requirements for the Kibana pods
```

Comment:
ResourceRequirements are not only limits, but also requests. AFAIK if you omit the description you will get the default description from corev1.ResourceRequirements type, which is a super description anyway. Of course you can copy it here and adapt it to explain things more for elasticsearch, e.g. the requests/limits for cpu/mem per kibana pod

Reply:
This is copied from https://github.com/openshift/elasticsearch-operator/blob/2a23f7cc1b904d9d34e00f7c84f671004eb1930c/manifests/4.7/elasticsearch-operator.v4.7.0.clusterserviceversion.yaml#L416 to reduce the output of `dyff`. Do you prefer omitting the description?

Reply:
I leave this up to you to improve the description here. I think we can tolerate if the final output of dyff has some differences due to improved documentation :rocket: